Basic/Basic/CreateEmployeeViewController.h

```diff
@@ -15,12 +16,13 @@
 
 @end
 
-@interface CreateEmployeeViewController : UIViewController
+@interface CreateEmployeeViewController : UIViewController<HSDatePickerViewControllerDelegate>
 
 @property (weak, nonatomic) id<CreateEmployeeDelegate> delagate;
 @property (weak, nonatomic) IBOutlet UITextField *firstName;
 @property (weak, nonatomic) IBOutlet UITextField *lastName;
 @property (weak, nonatomic) IBOutlet UITextField *salary;
 @property (strong, nonatomic) Employee *employee;
+@property (weak, nonatomic) IBOutlet UITextField *date;
```

Comment:
If a variable is named `date` I would expect it to be of type `NSDate`, but it isn't. Rename to dateTextField.